src/config.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Handles a configuration dictionary.
+ *
+ * @class Config
+ * @extends Model
+ */
+
+CKEDITOR.define( [ 'ckeditor', 'mvc/model', 'utils' ], function( CKEDITOR, Model, utils ) {
```

Comment:
Weird... I've created a test file with this:

```js
/* globals test */

'use strict';

test.example = function( test ) {
	return 1;
};
```

... and here jshint caught the unused parameter... we'll have to spend more time on this at some point... for now, I'm cleaning the params list, ofc.